src/components/Action/Action.vue

```diff
@@ -90,10 +89,12 @@ export default {
 	},
 	methods: {
 		toggleMenu() {
-			this.opened = !this.opened
+			this.open = !this.open
```

Comment:
You cannot change a prop. This is against the vuejs guidelines :(
You need a local clone of the value

Reply:
Yeah, I knew something felt wrong :)

Reply:
Fixed.